src/coreclr/tools/superpmi/superpmi/commandline.cpp

```diff
@@ -468,6 +473,17 @@ bool CommandLine::Parse(int argc, char* argv[], /* OUT */ Options* o)
                 }
                 o->hash = argv[i];
             }
+            else if ((_strnicmp(&argv[i][1], "streaming", argLen) == 0))
+            {
+                if (++i >= argc)
+                {
+                    LogError("'-streaming' must be followed by a file name or stdin.");
+                    // DumpHelp(argv[0]);
```

Comment:
Un-comment?

Reply:
I find the full help text is so verbose I lose track of the specific error.

Reply:
Uncommented it nonetheless.